src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - 50;
```

Comment:
Move number 50 to a variable so it's understandable what this number represents.